src/main/kotlin/no/nav/syfo/Apprec.kt

```diff
@@ -1,11 +1,43 @@
 package no.nav.syfo
 
+import java.time.LocalDateTime
 import no.nav.syfo.apprec.ApprecStatus
 import no.nav.syfo.model.ValidationResult
 
 data class Apprec(
-    val fellesformat: String,
+    val ediloggid: String,
+    val msgId: String,
+    val msgTypeV: String,
+    val msgTypeDN: String,
+    val genDate: LocalDateTime,
     val apprecStatus: ApprecStatus,
-    val tekstTilSykmelder: String?,
+    val tekstTilSykmelder: String? = null,
+    val senderOrganisasjon: Organisation,
+    val mottakerOrganisasjon: Organisation,
     val validationResult: ValidationResult?
 )
+
+data class Organisation(
+    val houvedIdent: Ident,
```

Comment:
Hva er hovedident? Er det orgnummer? Kan det i så fall hete orgnummer? :) Samme spm gjelder for helsepersonell også, er hovedident fnr eller en annen id? 

Og så mangler det en g i tilleggsIdenter begge steder, beklager pirking :P

Reply:
Ein houvedident, er den første og beste ident vi velger basert på det dei sender inn til oss.
Det kan være ein org, eller det kan være ein herid.
Her gjelder egentlig det samme, som over.

Pirk er bra, det er nå vi har tid og mulighet til å lage god kode